docs/history.md

```diff
@@ -1,25 +1,45 @@
 # Version history (from 2.0)
 
+## 2.8.0 (2024-05-08)
+
+- Release CloudNative.CloudEvents.Mqtt as 3.8.0, bumping the major
+  version to reflect adopting MQTTnet version 4.3.6.1152.
+  Fixes [#269](https://github.com/cloudevents/sdk-csharp/issues/269).
+- Other dependency updates:
+  - System.Text.Json: 5.0.2 => 8.0.4
```

Comment:
```suggestion
  - System.Text.Json: 5.0.2 => 8.0.4 (only used in netstandard2.0 and netstandard2.1 targets)
```

A bit pedantic, but I like the clarification that you provided for `System.Text.Encodings.Web` below. Maybe we can colocate the two under a separate heading?

Reply:
Yes, good plan, will do that tomorrow morning UK time. And more pedantry is always fine by me :) Thanks!

Reply:
Have updated, and moved the bit about targeting .NET 8 and using framework references where possible to the top, so that the conditionality makes more sense. Please take a look - docs are so important and I really appreciate your input :)